### ISSUE TITLE
FR: Histogram & Scatterplot

### ISSUE THREAD

**Issue by chrisgrieser** [alincoop/obsidian-tinychart#3]
By change, I just stumbled upon [this project](https://github.com/tammoippen/plotille?tab=readme-ov-file), which uses ascii characters to create scatterplots and histogramms. I think both would fit nicely with this plugin as well?

**Comment by alincoop** [comment]
Absolutely. Next up is line charts, but I'll add scatter plots and histograms to the list.
